datacenterlight/urls.py

```diff
@@ -7,9 +7,11 @@
 
 urlpatterns = [
     url(r'^$', IndexView.as_view(), name='index'),
-    url(r'^t$', IndexView.as_view(), name='index_t'),
-    url(r'^g$', IndexView.as_view(), name='index_g'),
-    url(r'^f$', IndexView.as_view(), name='index_f'),
+    url(r'^t/$', IndexView.as_view(), name='index_t'),
+    url(r'^g/$', IndexView.as_view(), name='index_g'),
+    url(r'^f/$', IndexView.as_view(), name='index_f'),
```

Comment:
Inconsistency for the win! Sorry for the confusion: let's just use "l" for linkedin and get rid of "fb", as we have "f" already.